packages/node/src/service-interfaces.ts

```diff
@@ -1,8 +1,8 @@
 import { Address } from "@counterfactual/common-types";
 
 export interface IMessagingService {
-  send(peerAddress: Address, msg: object);
-  receive(address: Address, callback: (msg: object) => void);
+  send(peerAddress: Address, msg: any);
```

Comment:
@cf19drofxots Why do we go with `any` here? I think it'd be nice if  `msg` had a well-defined type here.

Reply:
I left it loose here to not restrict usage and wanted to see what the use case pattern would be more concretely.

Will update to a more type-safe signature.